splunk-otel-android/src/main/java/com/splunk/rum/ActivityCallbacks.java

```diff
@@ -38,11 +39,15 @@
     private final Tracer tracer;
     private final VisibleScreenTracker visibleScreenTracker;
     private final AppStartupTimer startupTimer;
+    private final List<AppStateListener> appStateListeners;
+    //we count the number of activities that have been "started" and not yet "stopped" here to figure out when the app goes into the background.
+    private int numberOfOpenActivities = 0;
```

Comment:
Can this be a "normal" integer? No need for thread safety?

Reply:
It can be, since this is only ever accessed from the UI thread (which is where the callbacks run).